nearswap/tests/test_utils.rs

```diff
@@ -1,225 +1,196 @@
 // SPDX-License-Identifier: GPL-3.0-or-later
 // Copyright (C) 2020 Robert Zaremba and contributors
 
-use nearswap::util::*;
-use near_primitives::types::{AccountId, Balance};
+#![allow(unused)]
+
```

Comment:
let's:
* rename this file to `clp_utils.rs`
* move all not clp directly related things (eg `get_nep21_balance` , `show_nep21_bal
`) to other files (eg: `nep21_utils.rs`)

Reply:
ok